apps/client/src/layout/components/side-navigation/side-navigation.tsx

```diff
@@ -0,0 +1,40 @@
+import _SideNavigation from '@cloudscape-design/components/side-navigation';
+import { useIntl } from 'react-intl';
+
+import { useActiveHref } from './hooks/use-active-href';
+import { DASHBOARDS_HREF, ROOT_HREF } from '../../../constants';
+import { preventFullPageLoad } from '../../../helpers/events';
+import { useBrowser } from '../../../hooks/browser/use-browser';
+
+export function SideNavigation() {
+  const activeHref = useActiveHref();
+  const { navigate } = useBrowser();
+  const intl = useIntl();
+
+  return (
+    <_SideNavigation
```

Comment:
is underscore going to be our convention for CloudScape imports?

Reply:
I used the underscore because SideNavigation and TopNavigation are already great names. Elsewhere, the default imports are not renamed.